src/models/serializers.rs

```diff
@@ -130,6 +130,7 @@ pub(crate) fn extract_action_type_and_value_from_action_view(
                 "beneficiary_id": beneficiary_id,
             }),
         ),
+        ActionView::Delegate { .. } => (ActionKind::DelegateAction, json!({})),
```

Comment:
This is a missing implementation part.

Reply:
Thank you for providing the information, I will try to implement this part.

Reply:
Thanks! It means a lot, appreciate you helping the community of developers using the version we cannot support anymore! ❤️